DESIGN.md

```diff
@@ -353,6 +353,91 @@ These are done through the MediaWiki API (most likely through a bot account).
 See the [example](EXAMPLE.md) for a better demonstration of how one might
 upload these files.
 
+### Editing data
+
+As per [issue
+#32](https://github.com/opentechstrategies/torque/issues/32), users
+can edit data fields (e.g., to fix a typo in a city's name in the
+"City" field) if they have proper permissions.
+
+Because the original proposal data is inhaled from an on-disk
+spreadsheet at startup time and lives in an in-memory data structure,
+edits are persisted "off to the side" as overlays over that original
+data.  The overlay information is then consulted for updates when
+proposal data is loaded.
+
+For simplicity's sake, we store the edits in their own top-level data
+structure (pickled to and from disk just like the original data is).
+This is what that data structure looks like:
+
+```
+edits = {
+    "proposals": {
+        "1": {
+               "Application #":
+                  { "original": ORIGINAL_VALUE,
+                    "edits": [
```

Comment:
Note the suggested edits to our edit schema here @kfogel @frankduncan there was some discussion on this in channel but I'm not sure where we landed as to what we wanted (dict vs array) so I went with what is shown here.